src/Step/ValidatorStep.php

```diff
@@ -90,7 +90,10 @@ public function process($item, callable $next)
             $this->violations[$this->line] = $list;
 
             if ($this->throwExceptions) {
-                throw new ValidationException($list, $this->line);
+                $exception = new ValidationException($list, $this->line);
+                $this->line++;
```

Comment:
Not sure that's where line should be increased. This way you won't get the proper line number of the data row where the error happened, only an increasing sequence of numbers. If I understand well what the intent was here...

Reply:
Have refactored the way this is done 👍